cmd/tempo/app/modules.go

```diff
@@ -154,27 +153,32 @@ func (t *App) initQuerier() (services.Service, error) {
 	return t.querier, t.querier.CreateAndRegisterWorker(t.server.HTTP)
 }
 
+type cortexLimits struct{}
+
+func (cortexLimits) MaxQueriersPerUser(user string) int { return 0 }
```

Comment:
Is returning 0 here OK? Hope we won't be canceling all queries.

Reply:
This means that all queriers can be used for every tenant.  As far as I can tell, this matches previous behavior, and is related to the new shuffle-sharding improvements in cortex. We could add this to tenant overrides in the future if needed.